nss-day-cohort-29/nutshell-midnight-dream
# Nh fetch calls

# Description
This request is to update the code for the "tasks" application of our project. There is no new functionality but code has been commented and cleaned up for our team review. This is just an interim request to update my section for the group. 

## Type of change
Please delete options that are not relevant.


# Testing Instructions for Change Made
1. `git fetch --all`
2. `git checkout nh-fetch-calls`
3. Run grunt
4. create a new task- should pop up the task form
5. submit task- should send you back to the tasklist page
6. click the task name in the tasklist- should console log an id number (eventually edit functionality)

# Checklist:
